### DISCUSSION TITLE
엄격 모드가 자바스크립트 코드 작성 시 왜 중요한가요?

### DISCUSSION THREAD

**Discussion by lledellebell** [diving-javascript/you-dont-know-js-yet#61] [1. 스코프] [answered]
<details>
<summary>관련챕터</summary>
[Part2] Chapter1.5 런타임에 스코프 변경하기
</details>

**Comment by wogus216** [comment, marked as answer]
<details>
<summary>답변</summary>


엄격모드를 사용시 코드의 오류를 줄이고, 성능을 개선 또 안전한 코드를 작성하는데 도움이 됩니다.
</details>
